packages/suite/src/views/settings/general/SettingsGeneral.tsx

```diff
@@ -57,7 +55,7 @@ export const SettingsGeneral = () => {
             <SettingsSection title={<Translation id="TR_LABELING" />} icon="TAG_MINIMAL">
                 <Labeling />
                 {isMetadataEnabled &&
-                    (isProviderConnected ? (
+                    (isProviderConnected && metadata.enabled ? (
```

Comment:
additional check for `&& metadata.enabled` is useless here, there is `isMetadataEnabled` condition one line above

Reply:
f97d46d9bb